ush/wave_outp_cat.sh

```diff
@@ -105,15 +91,8 @@
         #rm $outfilefhr
       fi
     else
-      set +x
-      echo ' '
-      echo '************************************************************************** '
-      echo "*** FATAL ERROR : OUTPUT DATA FILE FOR BOUY $bouy at ${ymdh} NOT FOUND *** "
-      echo '************************************************************************** '
-      echo ' '
-      set_trace
-      err=2; export err;${errchk}
-      exit $err
+      echo "*** ERROR: OUTPUT DATA FILE FOR BOUY $bouy at ${ymdh} NOT FOUND *** "
+      exit 2
```

Comment:
```suggestion
      err_chk "FATAL ERROR: OUTPUT DATA FILE FOR BUOY $buoy at ${ymdh} NOT FOUND"
```

Reply:
I think we want to use `err_chk` at just the J-Job and ex-script level.  The ush scripts are fine to exit.

Reply:
Gotcha, ignore my other similar comments within ush scripts then. Thanks!

Reply:
I updated this comment to note the spelling mistake for buoy.